.gitignore

```diff
@@ -1 +1,2 @@
-node_modules/
\ No newline at end of file
+node_modules/
+.npm/
```

Comment:
Optional: you could re-add the trailing newline to make this diff less odd-looking. Trailing newlines are expected by Unix anyway https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
Oh, you _added_ the newline. My b

Reply:
Of course, auto newline in emacs man